test cases/common/238 fstrings/meson.build

```diff
@@ -0,0 +1,9 @@
+project('meson-test', 'c')
+
+n = 10
+m = 'bar'
+s = f'test {n} string ({n}): {m}'
+
+if s != 'test 10 string (10): bar'
+  error('Incorrect string formatting')
+endif
```

Comment:
You can use assert() instead.

Reply:
Oh yea, I keep forgetting we have assert().